safety/scan/command.py

```
@@ -1,9 +1,10 @@
+from collections import Counter
 from enum import Enum
 import itertools
```

Ruff

`itertools` imported but unused

Remove unused import: `itertools`

(F401)